app/screens/receive-bitcoin-screen/receive-bitcoin.tsx

```diff
@@ -100,6 +100,7 @@ const ReceiveBitcoinScreen = ({
                 {
                   text: LL.common.later(),
                   // todo: add analytics
+                  // eslint-disable-next-line no-console
                   onPress: () => console.log("Cancel/Later Pressed"),
```

Comment:
You can probably just remove this `console.log`

Reply:
will do. it was just triggering my code-check and preventing commits. but i will delete the console.log if its not needed